source/main/modules/window-manager/index.ts

```diff
@@ -207,18 +223,72 @@ export default class WindowManager {
   /**
    * Shuts down the window manager and performs final operations
    */
-  shutdown (): void {
+  shutdown(): void {
     this._persistWindowPositions()
   }
 
   /**
    * Listens to events on the main window
    */
-  private _hookMainWindow (): void {
+  private _hookMainWindow(): void {
     if (this._mainWindow === null) {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      if (process.platform == 'darwin') {
+        if (tray == null) {
+          let basepath = app.getAppPath()
+          let image = await nativeImage.createThumbnailFromPath(path.join(__dirname, 'assets/icons/128x128.png'), { width: 16, height: 16 })
+          tray = new Tray(image)
+          const contextMenu = Menu.buildFromTemplate([
+            {
+              label: 'Show Zettlr',
+              click: () => {
+                this.showAnyWindow()
+              },
+              type: 'normal'
+            },
+            { label: '', type: 'separator' },
+            {
+              label: 'Quit',
+              click: () => {
+                app.quit()
+              }, type: 'normal'
+            }
+          ])
+          tray.setToolTip('This is zettlr tray. \n click show Zettlr button to display app \n click quit to quit app')
+          tray.setContextMenu(contextMenu)
+        }
+      } else {
+        if (tray == null) {
+          tray = new Tray(path.join(__dirname, 'assets/icons/128x128.png'))
+          const contextMenu = Menu.buildFromTemplate([
+            {
+              label: 'Show Zettlr',
+              click: () => {
+                // Add show Zettlr window event to Windows #6
```

Comment:
Remove #6 because that is our team's issue number. Zettlr developers do not know about our team's issues.